Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/rdf/RDFMessageFactory.java

```diff
@@ -0,0 +1,147 @@
+
+                Resource resource = model.createResource(creator.getURI(entry));
+
+                resource.addProperty(identifiers, entry.getId());
+                resource.addProperty(madsrdfProperty, entry.getWord());
+                resource.addProperty(skosProperty, entry.getWord());
+                resource.addProperty(schemaName, entry.getWord());
+                resource.addProperty(schemaDescription, entry.getDescription());
+
+                if (entry.getConceptList() != null) {
+                    resource.addProperty(madsrdfCL, creator.getConceptListURI(entry));
+                }
+
+                if (entry.getCreatorId() != null) {
+                    resource.addProperty(csCreator, entry.getCreatorId());
+                }
+
+                if (entry.getModified() != null) {
+                    resource.addProperty(csCreator, entry.getCreatorId());
+                }
+
+                if (entry.getEqualTo() != null) {
+                    String[] equals = entry.getEqualTo().split(",");
+                    for (String equal : equals) {
+                        resource.addProperty(owlEqualTo, equal);
+                    }
+                }
+
+                if (entry.getSimilarTo() != null) {
+                    String[] similarTo = entry.getSimilarTo().split(",");
+                    for (String similar : similarTo) {
+                        resource.addProperty(rdfsSeeAlso, similar);
+                    }
+                }
+
+                if (entry.getWordnetId() != null) {
+                    resource.addProperty(owlEqualTo, creator.getWordnetURI(entry));
+                }
+
+                if (entry.getSynonymIds() != null) {
+                    String[] synonymIds = entry.getSynonymIds()
+                            .split(edu.asu.conceptpower.servlet.core.Constants.SYNONYM_SEPARATOR);
+                    for (String synonymId : synonymIds) {
+                        resource.addProperty(skosRelated, synonymId);
+                    }
+                }
+
+                Resource typeResource = model.createResource(skosType);
+                resource.addProperty(RDF.type, typeResource);
+
+                Resource typeRes = model.createResource(authorityType);
+                resource.addProperty(RDF.type, typeRes);
+
+                if (entry.getTypeId() != null) {
+                    Resource typesRes = model.createResource(creator.getTypeURI(entry));
+                    resource.addProperty(RDF.type, typesRes);
+                }
+
+                String matchesURI = entry.getTypeId();
+                if (matchesURI != null) {
+                    String matches = typeManager.getType(matchesURI).getMatches();
+                    Resource cdocResource = model.createResource(matches);
+                    resource.addProperty(RDF.type, cdocResource);
+                }
+                model.write(out, syntax);
+            }
+        } catch (BadURIException e) {
```

Comment:
why is it failing? what is the reason for that? Since all the URIs generated by conceptpower followed the above format, that would mean it would fail for almost all added concepts.